src/Models/Payload/Payload.php

```diff
@@ -22,8 +22,12 @@
 
 use Buckaroo\Models\AdditionalParameters;
 use Buckaroo\Models\ClientIP;
+use Buckaroo\Models\CustomParameters;
 use Buckaroo\Models\Model;
 
+/**
+ *
```

Comment:
remove comment

Reply:
Removed for now, but this doesn't really matter. It's auto-generated by IDE